src/main/resources/assets/terramap/lang/zh_tw.lang

```diff
@@ -194,19 +197,37 @@ terramap.terramapscreen.information.playergeo=玩家位置：%1$s° %2$s°
 terramap.terramapscreen.information.noplayer=玩家位置：-° -°
 terramap.terramapscreen.information.distortion=變形：%1$s 格/米；±%2$s°
 
-# # Marker controllers
+## Marker controllers
 terramap.terramapscreen.markercontrollers.buttons.mainplayer=切換主要玩家的可見性
 terramap.terramapscreen.markercontrollers.buttons.otherplayer=切換其他玩家的可見性
 terramap.terramapscreen.markercontrollers.buttons.animals=切換動物的可見性
 terramap.terramapscreen.markercontrollers.buttons.mobs=切換怪物的可見性
 terramap.terramapscreen.markercontrollers.buttons.direction=切換玩家朝向的可見性
 terramap.terramapscreen.markercontrollers.buttons.name=切換玩家名稱的可見性
 
-# # Layer panel
+## Layer panel
 terramap.terramapscreen.layerscreen.title=進階圖層設定
 terramap.terramapscreen.layerscreen.new=新圖層
+terramap.terramapscreen.layerscreen.raster_background.type=網格底圖
+terramap.terramapscreen.layerscreen.raster_background.no_offset=設置底圖投影偏移
+terramap.terramapscreen.layerscreen.raster_background.offset=底圖設定了投影偏移
+terramap.terramapscreen.layerscreen.generic.no_offset=設置圖層投影偏移
+terramap.terramapscreen.layerscreen.generic.offset=為這個圖層設定了投影偏移
+terramap.terramapscreen.layerscreen.generic.alpha=Alpha：
```

Comment:
Is it intended that alpha is not translated?

Reply:
Yes, at least that's the case for now